readthedocs/api/v3/proxied_urls.py

```diff
@@ -10,9 +10,12 @@
 from readthedocs.api.v3.proxied_views import ProxiedEmbedAPI
 from readthedocs.search.api.v3.views import ProxiedSearchAPI
 
+from .urls import router
+
 api_proxied_urls = [
     path("embed/", ProxiedEmbedAPI.as_view(), name="embed_api_v3"),
     path("search/", ProxiedSearchAPI.as_view(), name="search_api_v3"),
 ]
 
 urlpatterns = api_proxied_urls
+urlpatterns += router.urls
```

Comment:
I know Santos had some worried about handling modifications here. This seems like a pretty large change?

Santos mentioned:

> We can cache content over docs domains (assuming we only expose read-only resources, and on .org only).

And this doesn't seem to restrict any of those things? In particular, we need to remove auth from these API endpoints (https://github.com/readthedocs/readthedocs.org/blob/bb2aca3f4b877da3c3856a66bd2758cb7957074a/readthedocs/core/mixins.py#L25-L27), which this isn't doing?

Reply:
By the way, I understand we want auth here because .com will require authentication requests to return the correct data for those private projects.

Reply:
> By the way, I understand we want auth here because .com will require authentication requests to return the correct data for those private projects.

Yea, looks like we're setting `SessionAuthentication` explicitly on the Corporate side for the proxied APIs.